builds/ci/mqtt.yaml

```diff
@@ -1,19 +1,127 @@
-trigger: none
+trigger:
+  branches:
+    include:
+      - master
+      - iiot
 pr: none
 jobs:
-  ################################################################################
-  - job: compliance_amd64
-    ################################################################################
-    displayName: Compliance amd64
+
+################################################################################
+  - job: check_run_pipeline
+################################################################################
+    displayName: Check pipeline preconditions (changes ARE in builds or mqtt)
```

Comment:
why "ARE" in capital?

Reply:
Don't know :) copy-paste!

Reply:
I did it distinguish when task started but not supposed to.